datumaro/util/image.py

```diff
@@ -18,7 +17,7 @@ class _IMAGE_BACKENDS(Enum):
 _IMAGE_BACKEND = None
 _image_loading_errors = (FileNotFoundError, )
 try:
-    import cv2
+    importlib.import_module('cv2')
     _IMAGE_BACKEND = _IMAGE_BACKENDS.cv2
 except ImportError:
     import PIL
```

Comment:
Probably, `PIL` import also deserves a change.

Reply:
The `PIL` import is not unused, so I can't change it.